tests/unit/folio/coding/test_coding.py

```
@@ -196,6 +210,28 @@ async def r(_: NoComponents) -> Transition:
 
         assert focus.gate_answers == [False, True]
 
+    @staticmethod
+    def test_focus_options_ride_the_bundle_intact():
```

Ruff (0.16.0)

[warning] Missing return type annotation for staticmethod `test_focus_options_ride_the_bundle_intact`

Add return type annotation: `None`

(ANN205)